### ISSUE TITLE
Web - Step 5 - Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#74]
The SVG left, and right pointing arrows are missing a title, description, or some piece of ARIA as part of the Home Page, Previous Step, and Next Step links. Same applies to the bookmarks icon. There is no ALT text or some kind of description for the SVG images that are decorative before the "Step5 :.." text. They should at least have alt="" if nothing else. There are four other images that I couldn’t see without having my tools in IE to find that are empty or missing ALT text.

There might be missing ALT text on other pages that I didn't have time to check so please those too.

Step 5 - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/5

**Comment by KyleLivingstonDOL** [comment]
Thanks John. Can you please specify what you mean by "four other images"?

ZTI, can you please address the left and right pointing arrows?

**Comment by jfc3-dol** [comment]
I had to use the ANDI toolbar in Firefox or Chrome to test the pages because they would not display in IE at the time.

The four images refers the the icons in the footer.

**Comment by Abychalil** [comment]
Alt texts provided for all above mentioned fields..

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for adding the necessary ALT text. Will look at it when the next version/build is sent, and I have time to test.

**Comment by jfc3-dol** [comment]
There is now ALT text on the left pointing arrow and other items, so I'm going to close the issue.
